.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5507
+}
```

Comment:
[Optional] I recommend that you ignore the .vscode folder since it's only a local setting for your code editor and not required to run the project.